exorcist/models.py

```diff
@@ -1,23 +1,31 @@
 from enum import Enum
-from dataclasses import dataclass
-from datetime import datetime
+from typing import TypeVar, Generic
+import dataclasses
+
+# generics: the actual types here depend on the client library
+Result = TypeVar("Result")
+TaskDetails = TypeVar("TaskDetails")
 
 class TaskStatus(Enum):
+    """
+    Status of a given task.
+    """
     BLOCKED = 0
     AVAILABLE = 1
     IN_PROGRESS = 2
     RESULTS_READY = 3
```

Comment:
this status isn't planned any more right?

Reply:
removed in [240b325](https://github.com/OpenFreeEnergy/exorcist/pull/9/commits/240b3255297a57ba79900a462789673a14b58595)